### PR TITLE
Bug 2079673: disable migrated components

### PR DESCRIPTION
**Pull request by SanjalKatiyar**

### CODE REVIEW

**Comment by afreen23** [comment]
Lets merge it after enabling https://github.com/red-hat-storage/odf-console/pull/152

File: frontend/packages/ceph-storage-plugin/src/plugin.ts [hunk 1]
```diff
@@ -160,6 +166,9 @@ const plugin: Plugin<ConsumedExtensions> = [
           './components/create-storage-system/create-storage-system' /* webpackChunkName: "create-storage-system" */
         ).then((m) => m.default),
     },
+    flags: {
+      disallowed: [FEATURES.ODF_WIZARD],
+    },
```
Comment (SanjalKatiyar) [inline frontend/packages/ceph-storage-plugin/src/plugin.ts:171]: .

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**